nominal/sdk.py

```diff
@@ -458,6 +560,59 @@ def search_runs(
         for run in self._search_runs_paginated(request):
             yield Run._from_conjure(self, run)
 
+    def create_log_set(
+        self,
+        name: str,
+        timestamp_type: str,
+        description: str | None = None,
+        origin_metadata: Mapping[str, str] = MappingProxyType({}),
+    ) -> LogSetMetadata:
+        """
+        Creates a log set, to which logs can be attached using `attach-and-finalize`. The logs within a logset are
+        not searchable until the logset is finalized.
+
+        Timestamp type must be a string equal to either 'ABSOLUTE' or 'RELATIVE'.
+        """
+        timestamp_type_enum = None
+        if timestamp_type == "ABSOLUTE":
+            timestamp_type_enum = DataSourceTimestampType.ABSOLUTE
+        elif timestamp_type_enum == "RELATIVE":
+            timestamp_type_enum = DataSourceTimestampType.RELATIVE
+        else:
+            raise TypeError(f"timestamp type {timestamp_type} must be one of [RELATIVE, ABSOLUTE]")
```

Comment:
I also don't love taking in a string to do this check for abs vs relative but I didn't know what to do off the top of my head. In the API there's a type `UNKNOWN` though I thought to just throw for now.